app.rb

```diff
@@ -0,0 +1,131 @@
+require_relative './Classes/student'
+require_relative './Classes/teacher'
+require_relative './Associations/book'
+require_relative './Associations/rental'
+
+ACTIONS = {
+  1 => :list_books,
+  2 => :list_people,
+  3 => :create_person,
+  4 => :create_book,
+  5 => :create_rental,
+  6 => :list_rentals,
+  7 => :break
+}.freeze
+
+class App
+  def initialize
+    puts 'Welcome to School Library App!'
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def options
+    puts ''
+    puts 'Please choose an option by enterin a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
```

Comment:
- [ ] Please I am unable to list all rentals for a given person id, checkout the screenshot below and kindly fix the issue. 

![rental for a particular id](https://user-images.githubusercontent.com/65852621/226146103-7f99613e-a026-4fab-812f-a93296832517.png)